libstdc++-v3/configure

```diff
@@ -893,6 +893,7 @@ infodir
 docdir
 oldincludedir
 includedir
+runstatedir
```

Comment:
this is indicative of using the wrong version of autoreconf (some of the distro ones are patched - we need to use exactly what the GCC requirements are - or we end up with these spurious changes)